fastlane/Appfile

```diff
@@ -0,0 +1,11 @@
+app_identifier "com.mapbox.examples" # The bundle identifier of your app
+apple_id "zizi.miles@mapbox.com"  # Your Apple email address
```

Comment:
Is it possible to use `applemachine` account here?

Reply:
Absolutely will change and test.

Reply:
Hmm. When I attempt with the applemachine account, I receive an unauthorized access error. Perhaps we can set it to grab the account of whoever is releasing.